Source/Core/VideoBackends/OGL/OGLTexture.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "Common/GL/GLUtil.h"
+
+#include "VideoCommon/AbstractTexture.h"
+#include "VideoCommon/BPStructs.h"
+#include "VideoCommon/VideoCommon.h"
+
+namespace OGL
+{
+
+class OGLTexture : public AbstractTexture
+{
+public:
+
+  GLuint texId;
```

Comment:
These should be private and have default initializers, as well as the `m_` prefix.